sdcm/utils/url_validate.py

```diff
@@ -0,0 +1,50 @@
+import re
+
+import requests
+
+SUPPORTED_WEBSITES = ['s3.amazonaws.com', 'scratch.scylladb.com']
```

Comment:
please don't limit the places we could get repo from like that.

the idea is to check we could use that repo, not to limit it to specific hosts

Reply:
Yes, you're right, but to validate the URL I need to know the name of hosts.
For new URL we need to update this list.
What do you thinking?

Reply:
you can get if from the string you get... but, why is it important to you to know the name of hosts? what if a developer created a temporary repo to run tests for his tests?

Reply:
i don't understand what it means... the .repo or .list files could point anywhere and still be valid.. they are supposed to be repositories, and don't need to be hosted by scylla.com